wgsl/index.bs

```diff
@@ -14048,6 +14236,9 @@ Note: The result is not mathematically meaningful when `abs(t)` &ge; 1.
     <td>Description
     <td>Returns the cosine of `e`, where `e` is in radians.
     [=Component-wise=] when `T` is a vector.
+  <tr>
+    <td>Scalar [=domain=]
+    <td>Interval ([NINF], [PINF])
```

Comment:
There's a consistency problem here. Why list the full domain only sometimes?

Reply:
The full domain includes the infinities.  This one doesn't.

domains are over the extended reals.

That's another reason to use the word "interval" because in this case you don't want it to look like just a two-element set.

Reply:
no change made.